docs/ruleset_functions/get_number_of_floors.md

```diff
@@ -0,0 +1,54 @@
+# get_number_of_floors
+**Schema Version:** 0.0.23  
+
+**Description:** gets the number of floors in the building - logic is based on looking at the elevation of zones - zones with an elevation of <0 are considered underground and are not counted.  Parking Garages are not counted
```

Comment:
I think we need to count floors below grade. based on G3.1.1.1#2

Reply:
ok.  updated